src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -238,11 +199,10 @@ public function configureBuzz(ContainerBuilder $container, array $config)
     {
         $container->getDefinition('sonata.media.buzz.browser')
             ->replaceArgument(0, new Reference($config['buzz']['connector']));
-
         foreach ([
-            'sonata.media.buzz.connector.curl',
-            'sonata.media.buzz.connector.file_get_contents',
-        ] as $connector) {
+                     'sonata.media.buzz.connector.curl',
+                     'sonata.media.buzz.connector.file_get_contents',
+                 ] as $connector) {
```

Comment:
Why are you touching this?

Reply:
And why did you mark this as resolved?

Reply:
in the last commit, I'm reverting to last changes and add endpoint code only 
commit e76b3c56c6464323fc3118ca2e7976588b3a2df8

Reply:
Please revert that change too.